DragaliaAPI/Models/Database/Savefile/DbPlayerUserData.cs

```diff
@@ -76,6 +77,13 @@ public class DbPlayerUserData : IDbHasAccountId
 
     public int TutorialFlag { get; set; }
 
+    [NotMapped]
+    public ISet<int> TutorialFlagList
+    {
+        get => TutorialFlagUtil.ConvertIntToFlagIntList(TutorialFlag);
+        set => TutorialFlag = TutorialFlagUtil.ConvertFlagIntListToInt(value);
+    }
+
```

Comment:
ideally the repository should do conversions like this, that way it's more testable